core/src/main/java/uk/gov/gchq/koryphe/tuple/function/TupleAdaptedFunctionComposite.java

```diff
@@ -46,7 +46,7 @@ private Builder(final TupleAdaptedFunctionComposite<R> transformer) {
             this.transformer = transformer;
         }
 
-        public SelectedBuilder<R> select(final R[] selection) {
+        public SelectedBuilder<R> select(final R... selection) {
```

Comment:
This may causes problems in Gaffer as it will create a Object[] instead of R[]. Similar in other classes.